PaulasWeatherApp/PaulasWeatherApp/ViewController/ViewController.swift

```diff
@@ -8,48 +8,40 @@
 import UIKit
 
 class ViewController: UITabBarController {
-    var countryCityName: String?
-    var currentWeather: WeatherDetails?
-    var hourlyWeatherList = [WeatherDetails]()
-    var dailyWeatherList = [Daily]()
+    
+    let WeatherVC = CurrentWeatherViewController()
+    let MetricsVC = WeatherReportsViewController()
+    let SettingsVC = SettingsViewController()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
-        
-        let WeatherVC = CurrentWeatherViewController()
-        let MetricsVC = WeatherReportsViewController()
-        let SettingsVC = SettingsViewController()
-        
+    
         self.setViewControllers([WeatherVC ,MetricsVC ,SettingsVC ], animated: false)
         
         guard let items = self.tabBar.items else { return }
         
-        let images = ["clock", "calendar", "gear.circle"]
+        let images = ["clock", "calendar"]
         
-        for x in 0...2 {
+        for x in 0...1 {
```

Comment:
this for loop doesn't look good. You can do something like 
for tab in items { ... } instead